### PR TITLE
[DO NOT SQUASH MERGE] Converting related folder to markdown

### PR DESCRIPTION
**Pull request by Rumyra**
#### Summary

Converting related folder docs to markdown

#### Motivation

So we can move relevant ones to meta easily
NB #quicklinks sections cannot be converted

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by schalkneethling** [comment]
@Rumyra I see https://pr11273.content.dev.mdn.mozit.cloud/en-US/docs/Related/Project_guidelines reports two broken links, could we fix them with this PR?

**Review by schalkneethling — APPROVED** [review]
As the flaw system states, there are two broken links that we could perhaps fix. Other than that, it is an r+ from me

File: files/en-us/related/project_guidelines/index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+Have a think about how much work there is to do on your particular docs project, and how many people you might need for that.
+
+If it is a large project, you might benefit from having a few writers, a technical reviewer to check that the work is correct, a copy editor to clean up the language, someone to write code examples, etc.
+
+On a smaller project, you might have one or two people taking on multiple roles. Whatever you want to do is fine, as long as it works for you. You will be assigned a member of MDN staff to provide guidance on the MDN Web Docs side of things, and advice and help in general.
+
+You should assign one (or two) team leads, to act as a central point of contact for the MDN team member, and anyone else wishing to contact the team about it (you might get more offers of help, for example).
+
+Once your team is assembled, you'll need to get them all to [create MDN accounts](/en-US/docs/MDN/Contribute/Howto/Create_an_MDN_account), and then talk to your MDN rep to work out what site permissions each person needs (e.g. creating pages, moving pages, etc.)
```
Comment (ddbeck) [inline files/en-us/related/project_guidelines/index.md:32]: lol wiki

```suggestion
```

File: files/en-us/related/project_guidelines/index.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+- Team contact details, if you'd rather put them on a separate page.
+- Project-specific language or code guidelines.
+- Code templates to write new examples into.
+- Reusable asset lists.
+
+## Maintenance
+
+The documentation will need to be maintained to remain on MDN:
+
+- MDN is a Wiki; when others make changes to the docs, a core community member for that technology needs to review those changes to make sure the content is still good. This can be done via MDN's ["Watch" functionality](/en-US/docs/MDN/Contribute/Tools/Page_watching).
```
Comment (ddbeck) [inline files/en-us/related/project_guidelines/index.md:108]: lol wiki part 2

```suggestion
```

**Review by ddbeck — APPROVED** [review]
All this is going to get moved to the meta site sooner or later so I suggested just deleting the offending lines.  Feel free to merge, without or without the fixes for the broken links.

**Pull request merged**